frontend/src/styles/_uswds-theme.scss

```diff
@@ -48,6 +49,8 @@ in the form $setting: value,
     hover: false,
     visited: false
   ),
+  $height-settings: (responsive: true),
+  $z-index-settings: (responsive: true),
```

Comment:
Need responsiveness for z-index classes so that the nav bar is behind the overlay on small screens, but on larger screens the sticky header is above the content.